lib/Db/NoteMapper.php

```diff
@@ -2,6 +2,7 @@
 
 namespace OCA\NotesTutorial\Db;
 
+use OCA\NotesTutorial\AppInfo\Application;
```

Comment:
is it used, though? I think you can remove it

Reply:
Oupsy!

Reply:
Done